daemon/api_snap_conf.go

```diff
@@ -38,8 +38,8 @@ var (
 		Path:        "/v2/snaps/{name}/conf",
 		GET:         getSnapConf,
 		PUT:         setSnapConf,
-		ReadAccess:  authenticatedAccess{},
-		WriteAccess: authenticatedAccess{},
+		ReadAccess:  authenticatedAccess{Polkit: polkitActionManage},
+		WriteAccess: authenticatedAccess{Polkit: polkitActionManage},
```

Comment:
I'd suggest using a new action ID with a description that matches the type of access that is being granted. The new action should be defined in `data/polkit/io.snapcraft.snapd.policy`.

Also, what sort of user interface do you expect to see for this? If it was a toggle switch displaying whether AppArmor prompting was enabled that the user could turn on and off, authenticated read access means we'd show a polkit prompt just by displaying the UI before the user has decided to change anything. That seems like it'd be a bit confusing.

Reply:
For the read side we plan to separately add information about which experimental features are supported, respectively enabled to the /v2/system-info endpoint which is not restricted. cc @olivercalder